tests/test_collection.py

```diff
@@ -314,18 +317,29 @@ def test_collection_mapped():
     assert len(ls_ds[0]) == 2 and len(ls_ds[2]) == 2
 
     # adata3 goes first here
-    with collection_outer.mapped(label_keys="feat1", join_vars="auto") as ls_ds:
+    with collection_outer.mapped(label_keys="feat1", join="outer") as ls_ds:
         assert ls_ds.join_vars == "outer"
         assert len(ls_ds.var_joint) == 6
         assert len(ls_ds[0]) == 2
         assert len(ls_ds[0][0]) == 6
-        assert np.array_equal(ls_ds[1][0], np.array([4, 5, 8, 0, 0, 0]))
+        print(ls_ds[0][0])
```

Comment:
Are those print statements a relict from making the tests work?

Reply:
Yes, and a slight worry that this output might be helpful in the future again. It took me a while to understand the test, and so I thought I'd leave these statements there for a little longer.